keyboards/handwired/polykybd/split72/rules.mk

```diff
@@ -29,3 +29,6 @@ POINTING_DEVICE_DRIVER = cirque_pinnacle_i2c #POINTING_DEVICE_DRIVER = pimoroni_
 
 RAW_ENABLE = yes
 WPM_ENABLE = yes
+
+UNICODE_ENABLE = yes
```

Comment:
I'm not sure if we should enable unicode when using unicodemap... I have to read the QMK docs again.

Reply:
yeah, you're right, this is a mistake